### PR TITLE
issue-22-acknowledgement-footer

### LINKED ISSUE #22: Add "Acknowledgement of Country" in the Footer
We acknowledge and pay respects to the First Nations People of Australia, the land in which we live and work. we recognise the strength, resilience and capacity of aboriginal people and pay respects to elders past, present and emerging.

### PR DESCRIPTION
**Pull request by ghost**
## Rationale

Adds Acknowledgement to country to the footer.

## Issue Resolved

- Fixes #22

### CODE REVIEW

**Comment by Luukeeeee** [comment]
Hi @jabu-web , a simple issue, I think the "w" letter of "we" in the beginning of the second sentence should be capitalized.

File: src/component/footer/Footer.jsx [hunk 1]
```diff
@@ -1,83 +1,135 @@
...
+                    </div>
+                    {/* End Single Widget  */}
+
+                    <div className="col-lg-12">
+                      <div className="acknowledge-country">
+                        <br />
+                        <p>
+                          We acknowledge and pay respects to the First Nations
+                          People of Australia, the land in which we live and
+                          work. we recognise the strength, resilience and
```
Comment (QjStout) [inline src/component/footer/Footer.jsx]: Can you please capitalise the first letter of the the second sentence, i.e. "We recognise"

**Review by QjStout — CHANGES_REQUESTED** [review]
Looks good, just fix up that small typo and it's ready to rock 👍

File: src/component/footer/Footer.jsx (hunk 1 above)
Comment (ghost) [reply src/component/footer/Footer.jsx]: Will do

**Review by QjStout — APPROVED** [review]
LGTM 👍

**Pull request merged**